src/components/fx/index.js

```diff
@@ -45,6 +45,7 @@ module.exports = {
     unhover: dragElement.unhover,
 
     loneHover: require('./hover').loneHover,
+    multiHovers: require('./hover').multiHovers,
```

Comment:
🐄 this isn't new with your PR, but it's exacerbated here: we have multiple matching `require` statements - can you pull that out into one eg `var hoverModule = require('./hover');`?

Reply:
Done in 2100d96